aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -49,53 +67,60 @@ Toasts are compromised of three main elements:
 2. Content (Title + Message)
 3. Close button
 
-Aside from this, all toasts have a fixed width size at Medium (384px) while the height changes based off the amount of content. 
+All toasts have a fixed width size at Medium (384px) but the height can change based on the amount of content. 
 
 ### Status Indicator
 
-Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid this case for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
+Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid using toasts for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
 
-The status indicators used for toasts are strictly for Normal, Warning, and Unknown statuses.
+The status indicators used for toasts are restricted to Normal, Warning, Critical and Unknown statuses.  The crtical status should only be used to indicate a task completion that ended in an error and not for conveying critical or time-sensitive information to the user -- remember that a toast can be missed.
```

Comment:
```suggestion
The status indicators used for toasts are restricted to Normal, Warning, Critical and Unknown statuses.  The crtical status should only be used to indicate a task completion that ended in an error and not for conveying critical or time-sensitive information to the user since a toast can be missed.
```